Makefile

```diff
@@ -75,6 +76,19 @@ docker-tidy: $(DOCKER_CACHE)
 .linux-bin:
 	GOBIN=$(BINDIR) $(GO) install $(GOFLAGS) -ldflags '$(LDFLAGS)' github.com/vmware-tanzu/antrea/cmd/...
 
+# TODO: strip binary when building realses
+ANTCTL_BINARIES := antctl-darwin antctl-linux antctl-windows
```

Comment:
Even we do not want to include Windows and MAC bins now, we should include the Linux version to the container image. Does it mean we should build antctl with "make bin", and also have a way to build it with a Dockerfile (like other bins)?

Reply:
`github.com/vmware-tanzu/antrea/cmd/...` in `make bin` covers all executable targets include `antctl`.